examples/fillform.js

```diff
@@ -19,10 +18,9 @@ export default function() {
   page.$('input[name="password"]').type('123');
   page.$('input[type="submit"]').click();
 
-  sleep(5);
-  // Wait for next page to load
-  // FIXME: This doesn't work without the above sleep.
-  // Same with state 'domcontentloaded'.
+  // We expect the above form submission to trigger a navigation, so wait for it.
+  page.waitForNavigation();
+  // Wait for the next page to load
   page.waitForLoadState('networkidle');
```

Comment:
Do we still need this? `waitForNavigation` already listens for a `networkidle` event. I tested locally by running the script multiple times and there wasn't any error.

Reply:
Good point, it's not needed. Unless specified, `waitForNavigation()` defaults to waiting for a `load` event, which in this case serves the same purpose as `waitForLoadState('networkidle')`.

https://github.com/grafana/xk6-browser/blob/c8a58063eeb522ca999c9a6186057c5766221cd1/common/frame_options.go#L653-L659

Fixed in latest push.